internal/server/config.yaml

```
@@ -27,13 +27,8 @@ DB:
   NAME:
   PORT: 5432
   SSLMODE: disable
-NOTIFIER:
-  PROVIDER: slack
-  ACCESS_TOKEN:
-  WORKSPACES:
-    - WORKSPACE: goto
-      ACCESS_TOKEN:
-      CRITERIA: "email contains '@goto'"
+NOTIFIERS: "{\"gojek\":{\"provider\":\"lark\",\"client_id\":\"x\",\"client_secret\":\"x\"}}"
```

we can use `|` notation in yaml for better config readability + better example
```suggestion
NOTIFIERS: |
  {
    "my_lark": {
      "provider": "lark",
      "client_id": "",
      "client_secret": "",
      "messages": {}
    },
    "my_slack": {
      "provider": "slack",
      "access_token": "",
      "messages": {}
    }
  }
```